FirebaseStorage/Tests/Integration/FIRStorageIntegrationTests.m

```diff
@@ -117,8 +119,8 @@ - (void)setUp {
     }
 
     for (NSString *emptyFile in emptyFiles) {
-      FIRStorageReference *file = [[FIRStorage storage].reference child:emptyFile];
-      [file putData:[NSData data]
+      FIRStorageReference *file = [self.storage.reference child:emptyFile];
+        [file putData:[@"foo" dataUsingEncoding:NSUTF8StringEncoding]
```

Comment:
weird indent?

Reply:
This is just for Abe to do some debugging